app/src/test/java/io/gnosis/safe/utils/SemVerTest.kt

```diff
@@ -52,6 +65,43 @@ class SemVerTest {
         assertEquals(version4, range4.second)
     }
 
+    @Test
+    fun `parseRange (rangeString, ignoreExtensions) should return pair of SemVer`() {
+        val version1 = SemVer.parse("2.15.0")
+        val version2 = SemVer.parse("2.17.0")
+        val version3 = SemVer.parse("2.15.0-255-internal")
+        val version4 = SemVer.parse("2.17.0-300-internal")
+
+        val rangeString1 = "2.15.0...2.17.0"
+        val range1 = SemVer.parseRange(rangeString1, true)
+        assert(range1.first != null)
+        assert(range1.second != null)
+        assertEquals(version1, range1.first)
+        assertEquals(version2, range1.second)
+
+        val rangeString2 = "2.15.0"
+        val range2 = SemVer.parseRange(rangeString2, true)
+        assert(range2.first != null)
```

Comment:
https://stackoverflow.com/questions/2966347/assert-vs-junit-assertions 😛

Reply:
thank you for pointing this out